cmd/unused-exporter/exporter.go

```diff
@@ -133,11 +141,34 @@ func (e *exporter) Collect(ch chan<- prometheus.Metric) {
 					labels[k] = meta[k]
 				}
 				e.logger.Log("unused disk found", labels)
-				countByNamespace[meta["kubernetes.io/created-for/pvc/namespace"]] += 1
+				countByNamespace[meta.CreatedForNamespace()] += 1
 			}
 			for ns, c := range countByNamespace {
 				ch <- prometheus.MustNewConstMetric(e.count, prometheus.GaugeValue, float64(c), name, pid, ns)
 			}
+
+			for _, d := range disks {
+				m := d.Meta()
+
+				var ts float64
+				lastUsed := d.LastUsedAt()
+				if lastUsed.IsZero() {
+					ts = 0.0
+				} else {
+					ts = float64(lastUsed.UnixMilli())
+				}
```

Comment:
This is a really minor nit, I think this can be better expressed as:

```suggestion
ts := 0.0
lastUsed := d.LastUsedAt()
if !lastUsed.IsZero() {
    ts = float64(lastUsed.UnixMilli())
}				
```

Reply:
Yup, looks much nicer, although instead of `ts := 0.0` I think the original `var ts float64` is more readable.